backend/api.py

```diff
@@ -1,12 +1,54 @@
 # create our little application :)
-from flask import Flask
+import configparser
```

Comment:
Usually a nice convention to have all the 
`from x import y` together and then all the

`import z` together.

For this case:
```python
from flask import Flask, request
import configparser
import tweepy
```